src/environments/environment.prod.ts

```diff
@@ -1,4 +1,28 @@
+
 export const environment = {
   production: true,
-  configUrl: '/config'
+  login_url: 'http://localhost:3501/login',
```

Comment:
do we need this content here? how is this one different than the one in set-env.ts? thinking if we can remove this duplication with set-env.ts

Reply:
yes can be left empty, will do